docs/docs/FAQ.mdx

```diff
@@ -387,4 +387,20 @@ If your version of Immich is below 1.92.0 and the crash occurs after logs about
 
 Sometimes Immich logs errors such as "duplicate key value violates unique constraint" or "column (...) of relation (...) already exists". Because of Immich's container structure, this error can be seen when both immich and immich-microservices start at the same time and attempt to migrate or create the database structure. Since the database migration is run sequentially and inside of transactions, this error message does not cause harm to your installation of Immich and can safely be ignored. If needed, you can manually restart Immich by running `docker restart immich immich-microservices`.
 
+### Why does foreground backup stop when I navigate away from the app? Shouldn't it transfer the job to background backup?
+
+Foreground backup and background backup are two separate mechanisms. They don't communicate or interact with each other.
+
+Foreground backup is controlled by the user's action, while background backup is controlled by your phone's operating system. When the app is put in the background, the invocation of background tasks is delegated to the phone's operating system scheduler. It makes the call when the background task can run and how long it can run.
+
+The behaviors differ depending on your phone's manufacturer; most are under battery-saving policies.
+
+### Why is background backup on iOS not working?
+
+iPhone determines if a particular app can invoke background tasks by multiple factors, most of which we don't know and don't have control over. To increase the likelihood that background backup can run, follow the steps below:
+
+- Make sure you enable Background App Refresh in the phone's settings. `Settings > General > Background App Refresh`.
+- Disable Background App Refresh for apps that don't need background tasks to run. This will reduce the competition for background task invocation for Immich.
```

Comment:
```suggestion
- Disable Background App Refresh for apps that don't need background tasks to run.
  - This will reduce the competition for background task invocation for Immich.
```

Reply:
I personally like the current formatting more

Reply:
Makes sense. I was trying to divide "action steps" from "background details", but it's not strictly necessary for a short blurb like this.